CHANGELOG.md

```diff
@@ -1,3 +1,105 @@
+| bug fix |  **datepicker:** add role to date range input ([#19717](https://github.com/angular/components/issues/19717)) ([cfe37d1](https://github.com/angular/components/commit/cfe37d1)) |
+| bug fix |  **datepicker:** don't set aria-labelledby if form field does not have a label ([#19639](https://github.com/angular/components/issues/19639)) ([37e2171](https://github.com/angular/components/commit/37e2171)) |
+| bug fix |  **datepicker:** label state not being updated in some cases ([#19761](https://github.com/angular/components/issues/19761)) ([fcfc224](https://github.com/angular/components/commit/fcfc224)) |
+| bug fix |  **datepicker:** not marking as dirty when invalid value is typed in ([#19730](https://github.com/angular/components/issues/19730)) ([8934feb](https://github.com/angular/components/commit/8934feb)), closes [#19726](https://github.com/angular/components/issues/19726) |
+| bug fix |  **datepicker:** range input separator not hidden in high contrast mode ([#19706](https://github.com/angular/components/issues/19706)) ([9af441d](https://github.com/angular/components/commit/9af441d)) |
+| bug fix |  **dialog:** restore focus with the proper focus origin ([#9257](https://github.com/angular/components/issues/9257)) ([cf9bb1f](https://github.com/angular/components/commit/cf9bb1f)), closes [#8420](https://github.com/angular/components/issues/8420) |
+| bug fix |  **drag-drop:** allow handle to be set on drag element if there is a different drag root ([#19559](https://github.com/angular/components/issues/19559)) ([2335571](https://github.com/angular/components/commit/2335571)), closes [#19555](https://github.com/angular/components/issues/19555) |
+| bug fix |  **drag-drop:** error when cloning 0x0 canvas ([#19707](https://github.com/angular/components/issues/19707)) ([3614a0b](https://github.com/angular/components/commit/3614a0b)) |
+| bug fix |  **drag-drop:** not starting auto scroll when inside boundary ([#19865](https://github.com/angular/components/issues/19865)) ([39d48c7](https://github.com/angular/components/commit/39d48c7)), closes [#18596](https://github.com/angular/components/issues/18596) |
+| bug fix |  **drag-drop:** placeholder element not available in started event ([#19493](https://github.com/angular/components/issues/19493)) ([bc88cc1](https://github.com/angular/components/commit/bc88cc1)), closes [#19457](https://github.com/angular/components/issues/19457) |
+| bug fix |  **drag-drop:** sometimes incorrectly swapping items at the ends of the list ([#19633](https://github.com/angular/components/issues/19633)) ([0a6e4d6](https://github.com/angular/components/commit/0a6e4d6)), closes [#19249](https://github.com/angular/components/issues/19249) |
+| bug fix |  **drag-drop:** update free drag position on scroll ([#19648](https://github.com/angular/components/issues/19648)) ([9d903e0](https://github.com/angular/components/commit/9d903e0)) |
```

Comment:
All of these `drag-drop` issues should be listed in `cdk` rather than `material`.